<launch>
    <node name="ttyUSB1_manager" pkg="wubble_controllers" type="manager.py" output="screen" required="true">
        <param name="port_name" type="str" value="/dev/ttyUSB1"/>
        <param name="baud_rate" type="int" value="1000000"/>
        <param name="max_motor_id" type="int" value="25"/>
        <param name="update_rate" type="int" value="5"/>
    </node>
</launch>
